src/graphql/account.rs

```diff
@@ -301,7 +301,25 @@ impl AccountMutation {
         Ok(username)
     }
 
-    /// Authenticates with the given username and password
+    /// Authenticates with the given username and password.
+    ///
+    /// If the `lastSigninTime` value of the `account` is `None`, the operation will fail, and
+    /// it should be guided to call `signInWithNewPassword` GraphQL API.
+    ///
+    /// If the `signIn` GraphQL API response is the error message "a password change is
+    /// required to proceed", you can call the `signInWithNewPassword` GraphQL API.
+    ///
+    /// ```ignore
+    /// r#"mutation {
+    ///     signInWithNewPassword(username: "user", password: "pw", newPassword: "pw2") {
+    ///         token
+    ///     }
+    /// }"#
+    /// ```
```

Comment:
We don't need to present the signatures of GraphQL APIs in the code because users can not only infer them on their own but also check them in the GraphQL playground.

Reply:
I'll delete it.